operator/pom.xml

```diff
@@ -32,11 +32,10 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.operator.sdk.version>3.0.4</quarkus.operator.sdk.version>
-        <quarkus.version>2.7.3.Final</quarkus.version>
+        <quarkus.operator.sdk.version>3.0.6</quarkus.operator.sdk.version>
+        <quarkus.version>2.7.5.Final</quarkus.version>
         <quarkus.container-image.group>keycloak</quarkus.container-image.group>
-        <quarkus.jib.base-jvm-image>eclipse-temurin:11</quarkus.jib.base-jvm-image>
-        <quarkus.kubernetes.image-pull-policy>Never</quarkus.kubernetes.image-pull-policy>
+        <quarkus.jib.base-jvm-image>registry.access.redhat.com/ubi8/openjdk-11-runtime</quarkus.jib.base-jvm-image>
```

Comment:
Before implementing this I spoke with @stianst and this is the same way we are doing it for the main official Keycloak Docker image.

I do believe that this approach is ok, but I'm open to change in case I'm asked for.

Reply:
Feel free to discuss it with @stianst and let me know if I have to change something here